include/kep3/planet.hpp

```diff
@@ -146,6 +146,24 @@ struct planet_iface<void, void> {
 kep3_DLL_PUBLIC double period_from_energy(const std::array<double, 3> &, const std::array<double, 3> &, double);
 kep3_DLL_PUBLIC std::array<double, 6> elements_from_posvel(const std::array<std::array<double, 3>, 2> &, double,
                                                            kep3::elements_type);
+template <typename T>                                                        
+kep3_DLL_PUBLIC std::vector<double> default_eph_vectorization(T *self, const std::vector<double> &mjd2000s) {
```

Comment:
No need for ``kep3_DLL_PUBLIC`` here as this is a template, it will be compiled each time it is used.

Also, ``T *self`` should probably be ``const T *self`` for clarity (assuming ``eph()`` is always going to be a const method of ``self``).